VideoLocker/AndroidManifest.xml

```diff
@@ -217,6 +226,37 @@
         <service android:name="org.edx.mobile.services.DownloadSpeedService" />
         <service android:name="org.edx.mobile.services.FetchCourseFriendsService"/>
 
+
+        <!--   Parse Setup -->
+        <!--  https://www.parse.com/apps/quickstart#parse_push/android/existing  -->
+        <service android:name="com.parse.PushService" />
```

Comment:
Is there some way to do this programatically? (will Context.registerReceiver() do that?). We need to plan for the case where someone doesn't want to be using parse (like for white label)